universal-application-tool-0.0.1/conf/application.conf

```diff
@@ -386,3 +386,4 @@ ebean.default = "models.*"
 
 aws.s3.region=${?AWS_S3_REGION}
 aws.s3.bucket=${?AWS_S3_BUCKET_NAME}
+aws.local.endpoint="http://localhost:4566"
```

Comment:
Suggestion (blocking): I suspect this needs to be `http://localstack:4566`.

Reply:
Oh interesting. Is it because of the docker networking going on it looks for services as urls instead?

Reply:
yup, it uses the container names in docker-compose.yml as hostnames.